app/views/movement_confirmation_page.scala.html

```diff
@@ -15,48 +15,35 @@
  *@
 
 @import forms.ConsignmentReferences
-@import models.cache.JourneyType
 @import models.cache.JourneyType.JourneyType
-@import forms.ConsignmentReferenceType
 @import views.Title
-@import views.html.templates.main_template
-
-@this(main_template: main_template)
-
+@import components.gds.gds_main_template
+@import components.gds.pageTitle
+@import uk.gov.hmrc.govukfrontend.views.html.components.govukInsetText
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.insettext.InsetText
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+@import views.html.components.gds.link
+
+@this(
+    govukLayout: gds_main_template,
+    pageTitle: pageTitle,
+    govukInsetText: govukInsetText
+)
+
+@* TODO remove consignmentReferences: ConsignmentReferences when all pages are done *@
```

Comment:
Why would you like to remove consignmentReferences? The new design does not display it on the page?

Reply:
No, it is no longer required by the new design.